db_init.yaml

```diff
@@ -1,18 +1,23 @@
 # Use postgres/example user/password credentials
-version: '3.1'
-
 services:
+  adminer:
+    image: adminer
+    restart: always
+    ports:
+      - 8080:8080
+
   db:
     image: postgres
     restart: always
     environment:
       POSTGRES_PASSWORD: example
-      POSTGRES_DB: test
+      POSTGRES_USER: postgres
+      POSTGRES_DB: neighborhood
```

Comment:
I'm guessing that since you've removed the 'Create db ' line from the schema.sql file, all the other commands from schema.sql are executed directly on this `neighborhood` default database, right?